python_examples/face_recognition.py

```diff
@@ -114,6 +114,28 @@
         # In particular, a padding of 0.5 would double the width of the cropped area, a value of 1.
         # would triple it, and so forth.
 
+        # There is another overload of compute_face_descriptor that can take
+        # as an input aligned image of size 150x150. 
+        #
+        # Here is a sample usage of that
+
+        print("Computing descriptor on aligned image ..")
+        
+        # We will first generate the aligned image
+        face_chip = dlib.get_face_chip(img, shape)        
+
+        assert face_chip.shape[0] == face_chip.shape[1]
+        assert face_chip.shape[0] == 150        
+
+        # Now we simply pass this chip to the api
+        face_descriptor_from_prealigned_image = facerec.compute_face_descriptor(face_chip)                
+        print(face_descriptor_from_prealigned_image)        
+        
+        # Some checks to ensure that the descriptor is same whether you pass
+        # an aligned image or ask the api to perform the alignment
+        assert len(face_descriptor_from_prealigned_image) == len(face_descriptor)        
```

Comment:
Remove these asserts.  This kind of testing should be done in the unit tests in tools/python/test instead.

Reply:
ok